backend/model/db.js

```diff
@@ -59,6 +64,18 @@ Follow.belongsTo(User, {as: 'followed', foreignKey: 'follower_user_id', onDelete
 User.hasMany(Follow, {as: 'following', foreignKey: 'followed_user_id', onDelete: 'CASCADE'})
 Follow.belongsTo(User, {as: 'following', foreignKey: 'followed_user_id', onDelete: 'CASCADE'})
 
+User.hasMany(Event, {as: 'eventUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+Event.belongsTo(User, {as: 'eventUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+
+Tag.hasMany(EventTag, {as: 'eventTag', foreignKey: 'tag', onDelete: 'CASCADE'})
+EventTag.belongsTo(Tag, {as: 'eventTag', foreignKey: 'tag', onDelete: 'CASCADE'})
+
+Event.hasMany(EventTag, {foreignKey: 'id', onDelete: 'CASCADE'})
+EventTag.belongsTo(Event, {foreignKey: 'id', onDelete: 'CASCADE'})
+
+User.hasMany(Notification, {as: 'notificationUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+Notification.belongsTo(User, {as: 'notificationUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+
```

Comment:
And i think Notification relationships should be commented out since notification.js will be deleted.

Reply:
i updated relationships, now they're okay.